test/data.test.js

```diff
@@ -357,7 +357,9 @@ it("test_exponential_notation", () => {
     dataset.ImagePositionPatient[2] = 7.1945578383e-5;
     const buffer = data.write();
     const copy = dcmjs.data.DicomMessage.readFile(buffer);
-    expect(JSON.stringify(data)).toEqual(JSON.stringify(copy));
```

Comment:
Did we lose this test somehow?

Reply:
This did fail after my changes but it's behaving as expected since the test modifies the dataset and then re-writes it (`_rawValue` on `ImagePositionPatient` no longer matches since `Value` was modified prior to write).

So instead of expecting exact equality between original/new I just verify `ImagePositionPatient` tag. Based on the name `"test_exponential_notation"` I think that was the main spirit of this test.